source/include/rtcp_data_types.h

```diff
@@ -0,0 +1,385 @@
+ * +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+ <--+
+ * |            First        |        Number           | PictureID |    |
+ * +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+    | n SLIs.
+ * |            First        |        Number           | PictureID |    |
+ * +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+ <--+
+ */
+typedef struct RtcpSliPacket
+{
+    uint32_t senderSsrc;
+    uint32_t mediaSourceSsrc;
+    uint32_t * pSliInfos;
+    size_t numSliInfos;
+} RtcpSliPacket_t;
+
+#define RTCP_SLI_INFO_FIRST_BITMASK         0xFFF80000
+#define RTCP_SLI_INFO_FIRST_LOCATION        19
+
+#define RTCP_SLI_INFO_NUMBER_BITMASK        0x0007FFC0
+#define RTCP_SLI_INFO_NUMBER_LOCATION       6
+
+#define RTCP_SLI_INFO_PICTURE_ID_BITMASK    0x0000003F
+#define RTCP_SLI_INFO_PICTURE_ID_LOCATION   0
+
+#define RTCP_SLI_INFO_EXTRACT_FIRST( sliInfo )  \
+    ( ( ( sliInfo ) & RTCP_SLI_INFO_FIRST_BITMASK ) >> RTCP_SLI_INFO_FIRST_LOCATION )
+
+#define RTCP_SLI_INFO_EXTRACT_NUMBER( sliInfo ) \
+    ( ( ( sliInfo ) & RTCP_SLI_INFO_NUMBER_BITMASK ) >> RTCP_SLI_INFO_NUMBER_LOCATION )
+
+#define RTCP_SLI_INFO_EXTRACT_PICTURE_ID( sliInfo ) \
+    ( ( ( sliInfo ) & RTCP_SLI_INFO_PICTURE_ID_BITMASK ) >> RTCP_SLI_INFO_PICTURE_ID_LOCATION )
+
+/*-----------------------------------------------------------*/
+
+/*
+ * Receiver Estimated Max Bitrate (REMB):
+ *
+ * RFC - https://datatracker.ietf.org/doc/html/draft-alvestrand-rmcat-remb-03#section-2.2
+ *
+ *  0                   1                   2                   3
+ *  0 1 2 3 4 5 6 7 8 9 0 1 2 3 4 5 6 7 8 9 0 1 2 3 4 5 6 7 8 9 0 1
+ * +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+ * |V=2|P| FMT=15  |   PT=206      |             length            |
+ * +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+ * |                  SSRC of packet sender                        |
+ * +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+ * |                  SSRC of media source                         |
+ * +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+ * |  Unique identifier 'R' 'E' 'M' 'B'                            |
+ * +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+ * |  Num SSRC     | BR Exp    |  BR Mantissa                      |
+ * +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+ <--+
+ * |   SSRC feedback                                               |    |
+ * +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+    | n SSRCs.
+ * |   SSRC feedback                                               |    |
+ * +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+ <--+
+ */
+typedef struct RtcpRembPacket
+{
+    uint32_t senderSsrc;
+    uint32_t mediaSourceSsrc;
+    uint32_t bitRateMantissa;
+    uint8_t bitRateExponent;
+    uint32_t * pSsrcList;
+    size_t ssrcListLength;
```

Comment:
We can make `ssrcListLength` a `uint8_t` since it is 8 bits.

Reply:
Nice catch , let me update this.